bank_account.rb

```diff
@@ -0,0 +1,92 @@
+require 'date'
+
+class BankAccount
+  attr_accessor :name, :job, :email, :address, :pin, :balance, :last_withdrawal_date, :remaining_daily_withdrawal_limit
+  
+  WITHDRAWAL_LIMIT_PER_DAY = 5000
+
+  def initialize(name, job, email, address)
+    @name = name
+    @job = job
+    @email = email
+    @address = address
+    @pin = nil
```

Comment:
you don't need to initialize instance variables with nil, since their default value is nil